common/lockfree_queue.h

```diff
@@ -561,37 +559,33 @@ class RingChannel : public QueueType {
     using QueueType::read_available;
     using QueueType::write_available;
 
-    /**
-     * @brief Construct a new Ring Channel object
-     *
-     * @param busy_yield_timeout setting yield timeout, default is template
-     * parameter DEFAULT_BUSY_YIELD_TIMEOUT. Ring Channel will try busy yield
-     * in `busy_yield_timeout` usecs.
-     */
-    RingChannel(uint64_t busy_yield_turn = 64,
```

Comment:
try to keep interface backward compatible

Reply:
A fix has been added to make old interface keeps exactly the same as it used to be.

https://github.com/alibaba/PhotonLibOS/pull/347/commits/56673a961dda59a9d38a91fabe8db9fa2bb5568a